ocaml/xapi/xapi_cluster.ml

```diff
@@ -69,13 +69,23 @@ let create ~__context ~network ~cluster_stack ~pool_auto_join ~token_timeout ~to
 
 let destroy ~__context ~self =
   let dbg = Context.string_of_task __context in
-  assert_cluster_has_one_node ~__context ~self;
```

Comment:
I think this was the only use of this function, so I think we can remove it for now.

Reply:
```
~/s/xen-api (feature/REQ477/master|✔) # rg assert_cluster_has_one_node
ocaml/xapi/xapi_clustering.ml
161:let assert_cluster_has_one_node ~__context ~self =

ocaml/xapi/xapi_cluster.ml
72:  assert_cluster_has_one_node ~__context ~self;
~/s/xen-api (feature/REQ477/master|✔) #
```